src/tools/rbd/Utils.cc

```diff
@@ -464,7 +470,9 @@ int get_image_options(const boost::program_options::variables_map &vm,
   }
 
   opts->set(RBD_IMAGE_OPTION_ORDER, order);
-  opts->set(RBD_IMAGE_OPTION_FEATURES, features);
+  if (features_set) {
+    opts->set(RBD_IMAGE_OPTION_FEATURES, features);
+  }
```

Comment:
@dillaman  This makes `rbd clone` (when running without `--image-feature`) abort trying to get RBD_IMAGE_OPTION_FEATURES in `do_clone()` and failing:
```
tools/rbd/action/Clone.cc: 24: FAILED assert(r == 0)
```